content/articles/building-a-lyrics-search-app-with-vanilla-javascript/index.md

```diff
@@ -0,0 +1,463 @@
+
+#Building a Lyrics Search app with Vanilla Javascript
+
+![hero image](/building-a-lyrics-search-app-with-vanilla-javascript/hero.jpg)
+
+Knowing the lyrics of a song is a major problem most song-lovers encounter, especially when it is a rap song. Finding a solution to this problem, by creating a platform where users can search for lyrics by entering the artist name or title of the song is a lovely and useful project.
+
+In this article, I will walk you through how you can create a lyrics search app using HTML5, CSS3, Vanilla JS,async-await with fetch method, OVH API, and ES6 features like arrow functions.
+
+##Prerequisites
+..* A code editor. For example: Visual studio code.
+
+..* Some basic knowledge of HTML , CSS , and JAVASCRIPT.
+
+..* A laptop with an internet connection
+
+##Structuring The Lyrics Search App With Html5.
+
+ To begin, create a folder named `lyrics-app`, then you create three files in the folder:
+
+..*`lyrics.html`
+..*`lyrics.css`
+..*`lyrics.js`
```

Comment:
Kindly use the correct list structure

Though, I'm not sure whether or not this is supposed to be a list 🤔